Gopkg.lock

```diff
@@ -356,12 +356,27 @@
 
 [[projects]]
   branch = "master"
-  digest = "1:6c3d8af938c1d4c2b73182d157faf50f98930058498b0de4ac8888618d8236c8"
+  digest = "1:8d0349ca7980bc787c8c595434618a5abc98076d07a50ca38c9a80685e70f80e"
   name = "github.com/openshift/api"
-  packages = ["config/v1"]
+  packages = [
+    "config/v1",
+    "route/v1",
+  ]
   pruneopts = "NUT"
   revision = "6a375c5bd37e0a30bf3da96935273fa41f957a0d"
 
+[[projects]]
```

Comment:
can you add corresponding section to `Gopkg.toml` for this dependency.

Reply:
/hold

For this

Reply:
sure, will do